barista-compose/src/main/java/com/alorma/barista_compose/assertion/BaristaAssertions.kt

```diff
@@ -0,0 +1,71 @@
+package com.alorma.barista_compose.assertion
+
+import androidx.annotation.StringRes
+import androidx.compose.ui.test.SemanticsNodeInteraction
+import androidx.compose.ui.test.SemanticsNodeInteractionCollection
+import androidx.compose.ui.test.assertAny
+import androidx.compose.ui.test.assertIsDisplayed
+import androidx.compose.ui.test.hasText
+import androidx.compose.ui.test.junit4.ComposeTestRule
+import androidx.compose.ui.test.junit4.createAndroidComposeRule
+import androidx.compose.ui.test.junit4.createComposeRule
+import androidx.compose.ui.test.onChildren
+import androidx.compose.ui.test.onNodeWithText
+import com.alorma.barista_compose.assertion.internal.resources
+
+fun ComposeTestRule.assertIsDisplayed(
```

Comment:
Hi! Thanks a lot for your PR! Following Barista naming, I would rename this function to `assertDisplayed`

Reply:
Done, thanks for suggestion